airframe-log/README.md

```diff
@@ -262,6 +262,13 @@ if(logger.isDebugEnabled) {
 Log message String generation will not happen unless the debug log is effective. 
 Scala macro is also used for finding source code location (LogSource).
 
+### To disable Logging at compile-time
+It is possible to disable the generation of logging statements at compile time.
+Add `scalacOptions += "-Xmacro-settings:wvlet.log.disable.[REPLACE_WITH_LEVEL]"` compiler option to `build.sbt` 
```

Comment:
The meaning of REPLACE_WITH_LEVEL is not clear without looking LocMacros code. Can we use a more clearer naming? For example, SUPPRESS_BELOW = {ERROR, WARN, INFO, DEBUG}: 

If SUPPRESS_BELOW = INFO, then DEBUG, TRACE log code generation will be suppressed.

Reply:
Thanks for the feed back. Is it OK now?

Reply:
Yes. LGTM now